manifests/params.pp

```diff
@@ -5,6 +5,7 @@
   $version             = 'present'
   $user                = 'puppet'
   $group               = 'puppet'
+  $ip                  = '0.0.0.0'
```

Comment:
I don't know if IPv6 is enabled by default with PL packages, but this would limit it to IPv4 only, correct?